openemory/publication/forms.py

```diff
@@ -215,11 +222,12 @@ class UploadForm(forms.Form):
                   'This is required to submit an article.',
         error_messages={'required': 'You must indicate assent to upload an article'},
         widget=forms.CheckboxInput(attrs={'class': 'outline'}))
-    pdf = forms.FileField(label='Upload PDF',
+    content_model = forms.ChoiceField(choices= _ContentChoices, required=True,label='Type of Material',help_text='Please choose the option that best represents your work')
+    pdf = forms.FileField(label='File Upload',
          # customize default required message ('this field is required')
          error_messages={'required': 'A PDF file is required to submit an article.'},
-         widget=forms.FileInput(attrs={'class': 'text'}),
-         validators=[FileTypeValidator(types=['application/pdf'],
+         widget=forms.FileInput(attrs={'class': 'text', 'accept': '.pdf,.doc,.docx,.jpeg,.png,.ppt,.pptx,.tiff,.xls,.xlsx'}),
+         validators=[FileTypeValidator(types=['application/pdf', 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'application/msword', 'application/vnd.openxmlformats-officedocument.presentationml.presentation', 'application/vnd.ms-powerpoint', 'image/jpeg', 'image/tiff', 'image/png', 'application/vnd.ms-excel', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'],
                                        message=PDF_ERR_MSG)])
```

Comment:
Does this error message need to change too?

Reply:
Fixed